js/stateless.js/src/state/types.ts

```diff
@@ -7,25 +7,57 @@ export enum TreeType {
     /**
      * v1 state merkle tree
      */
-    State = 0,
+    StateV1 = 1,
     /**
      * v1 address merkle tree
      */
-    Address = 1,
+    AddressV1 = 2,
     /**
      * v2 state merkle tree
      */
-    BatchedState = 2,
+    StateV2 = 3,
     /**
      * v2 address merkle tree
      */
-    BatchedAddress = 3,
+    AddressV2 = 4,
 }
 
-export type ActiveTreeBundle = {
+/**
+ * Public keys for a state tree, versioned via {@link TreeType}. The protocol
+ * stores compressed accounts in state trees.
+ *
+ * Onchain Accounts are subject to Solana's write-lock limits.
+ *
+ * To load balance transactions, use {@link pickStateTreeInfo} to
+ * select a random tree from active Trees.
+ *
+ * Example:
+ * ```typescript
+ * const stateTreeInfos = await getCachedActiveStateTreeInfos();
+ * const randomStateTreeInfo = pickStateTreeInfo(stateTreeInfos);
+ * const ix = CompressedTokenProgram.compress({
+ *     ... // other params
+ *     outputStateTreeInfo: randomStateTreeInfo
+ * });
+ * ```
+ */
+export type StateTreeInfo = {
+    /**
+     * Account containing the Sparse Merkle tree in which a compressed
+     * account is stored.
+     */
     tree: PublicKey;
+    /**
+     * The state nullfier queue belonging to merkleTree.
+     */
     queue: PublicKey | null;
+    /**
+     * The compressed cpi context account.
+     */
     cpiContext: PublicKey | null;
```

Comment:
can we make these non optional?
```suggestion
    queue: PublicKey;
    /**
     * The compressed cpi context account.
     */
    cpiContext: PublicKey;
```